bin/live/pycbc_live_combine_single_fits

```diff
@@ -166,16 +167,20 @@ for ifo in args.ifos:
         # Sort alpha and counts by date
```

Comment:
I just noticed that `u` and `l` appear to be unused. It there any reason to include `bu` and `bl` in the zip? Could  you do instead
```python
for counter, (a, c) in enumerate(zip(alphas_bin[ifo], counts_bin[ifo])):
```

Reply:
I didn't realise you could unpack like that either, that makes things much nicer